src/content/getting-started/cloud-provider-accounts/cluster-api/azure/index.md

```diff
@@ -0,0 +1,141 @@
+
+2. An individual has to have following permissions and tools working within the designated Azure subscription:
+
+* An account with [Owner](https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#owner) or [User Access Administrator](https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#user-access-administrator) role.
+* [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli) installed.
+
+### Configure subscription to allow access to Giant Swarm Support
+
+First necessary step of configuring the Azure environment is to grant access for Giant Swarm Ops/Support to your subscription in order to provide 24/7 support. Access to the portal is important part of the provided support, where in some cases manual interventions have to take place.
+Easiest way is to use the Azure Lighthouse service, that allows to delegate the management of resources to third parties. While following this part of the guide, you will allow the Giant Swarm Staff group to manage your resources. This is beneficial as you will not have to manage access for each person separately within your subscription, but you will be adding a managed group that is kept up to date with the current active Giant Swarm Staff from our side.
+
+We require a built in role `Contributor` to access the resources that Giant Swarm is deploying and it can be used by default from the [Azure RBAC](https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles).
+
+You can also create your own role assignment with restrictions to access specific Resource Groups, however it needs to be remembered to refresh the role with every newly created cluster so we can provide full support starting from the clusters creation.
+
+When all is set you can simply run this command:
+
+```nohighlight
+az deployment create --name <deploymentName (unique by subscription)> \
+                     --location <AzureRegion> \
+                     --template-file delegatedResourceManagement.json \
+                     --parameters delegatedResourceManagement.parameters.json \
+                     --verbose
+```
+
+You will have to supply a general Delegated Resource Management [template file](https://raw.githubusercontent.com/giantswarm/azure-operator/master/docs/delegatedResourceManagement.json).
+
+The Delegated Resource Management template uses a [parameters file](https://raw.githubusercontent.com/giantswarm/azure-operator/master/docs/delegatedResourceManagement.parameters.json) to supply the needed variables for configuration.
+
+__Warning__: Please remember to change the `roleDefinitionId` in case you would like to use your custom role definition. Moreover, ask your Account Engineer so he can provide you with the `GiantSwarmPrincipalID` and `GiantSwarmTenantID` values.
```

Comment:
When _what_ is set? Above, there were no instructions to get anything "ready" or "set", apart from making a choice between built-in and custom role. There's no mention of a `deploymentName` and what is is, JSON parameter files, what `az deployment create` does, etc.

Let me try to rewrite this...

````suggestion
Please follow these instructions to create the Azure Deployment Environment:

* Download the template file [delegatedResourceManagement.json](https://raw.githubusercontent.com/giantswarm/azure-operator/master/docs/delegatedResourceManagement.json). This file is left unchanged.
* Download the parameters example file [delegatedResourceManagement.parameters.json](https://raw.githubusercontent.com/giantswarm/azure-operator/master/docs/delegatedResourceManagement.parameters.json) and fill in the required fields:

    * Replace `<GiantSwarmTenantID>` and `<GiantSwarmPrincipalID>` with the values provided by Giant Swarm's Account Engineer
    * Keep the value of `roleDefinitionId` if you are fine assigning the `Contributor` role. If a custom role is desired, please replace the value.
* Now you can run this command to create the Deployment Environment:

    ```sh
    az deployment create --name <deploymentName (unique by subscription)> \
                         --location <AzureRegion> \
                         --template-file delegatedResourceManagement.json \
                         --parameters delegatedResourceManagement.parameters.json \
                         --verbose
    ```
````

Plus follow-up https://github.com/giantswarm/roadmap/issues/2907 to replace the deprecated `az deployment create` command line.

This assumes that _Deployment Environment_ is really the right term. I don't have experience here.

**TODO:** `deploymentName` value 🤷‍♂️